cds_ils/importer/providers/ebl/rules/document.py

```diff
@@ -142,8 +144,13 @@ def alternative_identifiers(self, key, value):
 def languages(self, key, value):
     """Translates languages fields."""
     lang = clean_val("b", value, str).lower()
+    languages = self.get("languages")
```

Comment:
if the field is optional, you should have `[]` as default value:
```suggestion
    languages = self.get("languages", [])
```